tools/phpstan/src/ProhibitCastorFunctionInClass.php

```diff
@@ -8,6 +8,10 @@
 use PHPStan\Rules\Rule;
 use PHPStan\Rules\RuleErrorBuilder;
 
+if (class_exists(ProhibitCastorFunctionInClass::class)) {
+    return [];
+}
```

Comment:
small hack which allow to include this file into the phpstan configuration without declaring the autoloader on this.

Can be improved by providing more options in the remote lib, but right now this works fine